.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Might want to pin it to `ubuntu-20.04` so the world doesn't shift under us?

Reply:
Maybe later. For now, I'm happy for the world to shift so we find new world-shift-related issues sooner. It'll be a good mitigation when that time comes, though.